package.json

```diff
@@ -34,7 +35,10 @@
         "typescript": "~4.7.4"
     },
     "dependencies": {
-        "aws-cdk-lib": "^2.93.0"
+        "aws-cdk-lib": "^2.93.0",
+        "commander": "^12.0.0",
+        "enquirer": "^2.4.1",
+        "figlet": "^1.7.0"
```

Comment:
did you verify that these dependencies have the right licenses that we would be allowed to use?

Reply:
Where can I verify that? Also thanks for highlighting this, I need to remove `commander` since I ended up not using it

Reply:
They're both MIT licensed (assuming commander is being removed although that is also MIT).